src/view/items/partial/getPartialTemplate.js

```diff
@@ -14,6 +14,11 @@ export default function getPartialTemplate ( ractive, name, parentFragment ) {
 		// parse and register to this ractive instance
 		let parsed = parser.parseFor( partial, ractive );
 
+		// register extra partials on the ractive instance if they don't already exist
+		for ( let k in parsed.p ) {
+			if ( !( k in ractive.partials ) ) ractive.partials[ k ] = parsed.p[ k ];
+		}
+
```

Comment:
I did not know we had a `fillGaps` in util! I've pushed a change that cleans it up a little an (re-)introduces a use of `fillGaps`.

It looks like `Object.getOwnPropertyNames` would be the ideal replacement for the for-in, but it would need to be polyfilled in legacy alongside `Object.keys` for IE8.